examples/fluids/qfunctions/riemann_solver.h

```diff
@@ -84,13 +84,11 @@ CEED_QFUNCTION_HELPER StateConservative Flux_HLL_fwd(State left, State right, St
   UnpackState_U(dflux_left, dF_l);
   UnpackState_U(dflux_right, dF_r);
   for (int i = 0; i < 5; i++) {
-    const CeedScalar U_diff      = U_r[i] - U_l[i];
-    const CeedScalar S_diff      = S_r - S_l;
-    const CeedScalar F_hll_denom = S_r * F_l[i] - S_l * F_r[i] + S_l * S_r * U_diff;
+    const CeedScalar S_diff = S_r - S_l;
 
-    dF_hll[i] += ((F_l[i] + S_r * U_diff) * S_diff - F_hll_denom) / Square(S_diff) * dS_r;
-    dF_hll[i] += ((-F_r[i] + S_r * U_diff) * S_diff + F_hll_denom) / Square(S_diff) * dS_l;
-    dF_hll[i] += (S_r * dF_l[i] - S_l * dF_r[i] + S_r * S_l * dU_r[i] - S_r * S_l * dU_l[i]) / S_diff;
+    dF_hll[i] += (S_l * (-F_l[i] + F_r[i] + S_l * U_l[i] - S_l * U_r[i]) / Square(S_diff)) * dS_r;
+    dF_hll[i] += (S_r * (F_l[i] - F_r[i] - S_r * U_l[i] + S_r * U_r[i]) / Square(S_diff)) * dS_l;
+    dF_hll[i] += (S_r * dF_l[i] - S_l * dF_r[i] + S_r * S_l * (dU_r[i] - dU_l[i])) / S_diff;
```

Comment:
is this the part that had an error?

Reply:
There were two parts, both in the same file. So this is one of them.

Reply:
The other error (for `a_roe` in `ComputeHLLSpeeds_Roe_fwd`) is easy to spot the difference. TBH I'm not sure about the difference here. But the fix comes from `sympy` and allowed the test to pass (it wasn't before this fix).

Reply:
Cool, just wanted to make sure I was understanding which parts were test changes, convention changes, and bug fixes